tools/Altinn.Correspondence.DialogActivityExporter/test-export.ps1

```
@@ -0,0 +1,181 @@
+    $OutputPath = "C:\temp\test_export_$($Issue)_$($timestamp).csv"
+}
+
+# Ensure output directory exists
+$outputDir = Split-Path -Parent $OutputPath
+if (-not (Test-Path $outputDir)) {
+    New-Item -ItemType Directory -Path $outputDir -Force | Out-Null
+    Write-Host "Created output directory: $outputDir" -ForegroundColor Green
+}
+
+# Build command arguments
+$args = @(
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)